frontend/javascript/components/redaction/pdf-redaction.vue

```diff
@@ -648,8 +667,13 @@ export default {
         this.page = page
 
         let maxWidth = this.$refs.containerWrapper.offsetWidth
+        if (!this.$refs.containerWrapper) {
```

Comment:
This code will never be reached, as the line above would cause a `TypeError` should `this.$refs.containerWrapper` be null-ish.